app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -128,8 +128,21 @@ def lt_map_get_form_vars
   end
 
   def label_tag_mapping_add(entity, label_name, cat_description)
+    entity_str = ''
     prefix = ContainerLabelTagMapping::AUTOTAG_PREFIX
-    entity_str = entity.nil? ? "" : entity.underscore
+
+    # The entity is a string in the form "Provider::ResourceType".
+    if entity
+      prefix, entity = entity.split('::')
+      # Backwards compatibility
+      if prefix && entity
+        prefix.downcase!
```

Comment:
Consider just having lowercase prefixes in MAPPABLE_ENTITIES.
Capitalized with `::` looks like provider class name, but it's not exactly, IMO more confusing then helpful.

But then you might need upcase to display as "Amazon/Vm" in UI?  BTW, will we want translation of Amazon.
BTW, your screenshot shows "Amazon/Vm" with slash, where does `::` -> `/` replacement happen?